dart-2.12/pubspec.yaml

```diff
@@ -7,4 +7,4 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
       
 dev_dependencies:
-  pedantic: ^1.9.0
\ No newline at end of file
+  lints: ^2.0.0
```

Comment:
lints 2.0 requires Dart SDK 2.17 so this dart 2.12 starter shouldn't use lints 2.0.0